drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -344,16 +322,27 @@ def generic_upsert(file:, conflict_target:, klass:, ignore_columns: [])
 
       csv = records_from_csv(file)
 
-      columns_to_update = csv.headers - conflict_target - ignore_columns
+      # Fix known issues in CSV headers
+      header_fixes = {
+        'HousingAssessmemnt' => 'HousingAssessment',
+        'ZIP' => 'Zip',
+      }
```

Comment:
Fixing typos rather than requesting a change. It will also work if they use the correct format